### PR TITLE
Move websocktunnel into the monorepo

### LINKED ISSUE #2562: [websocktunnel-monorepo] migrate websocktunnel to the monorepo

### PR DESCRIPTION
**Pull request by djmitche**
More code!  This brings the history of websocktunnel along, but I did run `git-filter-branch` to strip out the `vendor/` directory to save space.  I think that's OK since it's not original content anyone created, and is redundant to `Gopkg.toml` anyway.

This also centralizes management of the TC version in `internal.Version`.

Github Bug/Issue: Fixes #2562

### CODE REVIEW

**Comment by imbstack** [comment]
this has a bunch of conflicts now. should I wait until it is rebased to review or just do it now?

**Comment by djmitche** [comment]
I'll rebase, just a minute..

**Comment by djmitche** [comment]
(I merged instead since rebase won't work with all that history)

File: tools/websocktunnel/.taskcluster.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
+version: 1
```
Comment (imbstack) [inline tools/websocktunnel/.taskcluster.yml:1]: we could rm this file

Comment (djmitche) [reply tools/websocktunnel/.taskcluster.yml:1]: thanks, nice spot

**Pull request merged**